instrumentation/opensearch/opensearch-java-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opensearch/java/v3_0/OpenSearchJavaRequest.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.opensearch.java.v3_0;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class OpenSearchJavaRequest {
```

Comment:
This comment applies to many of these classes, but I'm not sure including "Java" in the class names is necessary
```suggestion
public abstract class OpenSearchRequest {
```